launch/tier4_simulator_launch/launch/simulator.launch.xml

```diff
@@ -143,4 +143,11 @@
       <arg name="initial_engage_state" value="$(var initial_engage_state)"/>
     </include>
   </group>
+
+  <!-- traffic light visualizer -->
+  <group>
+    <include file="$(find-pkg-share traffic_light_visualization)/launch/traffic_light_map_visualizer.launch.xml">
+      <arg name="input/tl_state" value="/perception/traffic_light_recognition/traffic_signals"/>
+    </include>
+  </group>
```

Comment:
If possible, I would like to avoid writing in simulator launch.
The best would be to create it as an Rviz plugin, but is that hard?